receiver/hostmetricsreceiver/internal/scraper/processscraper/process.go

```diff
@@ -96,6 +96,7 @@ type processHandle interface {
 	PageFaults() (*process.PageFaultsStat, error)
 	NumCtxSwitches() (*process.NumCtxSwitchesStat, error)
 	NumFDs() (int32, error)
+	RlimitUsage(bool) ([]process.RlimitStat, error)
```

Comment:
Can we name the boolean parameter to this method and explain what it is used for? Right now it's unclear what its effect is.

Reply:
Added parameter name and the comment from gopsutil describing the parameter.